src/data/move-attrs/variable-move-type-chart-attr.ts

```diff
@@ -18,7 +18,13 @@ export class VariableMoveTypeChartAttr extends MoveAttr {
    * @param _defType A single defensive type of the target
    * @returns true if application of the attribute succeeds
    */
-  override apply(_user: Pokemon, _target: Pokemon, _move: Move, _multiplier: NumberHolder, _defType: Type): boolean {
+  override apply(
+    _user: Pokemon | null,
+    _target: Pokemon | null,
```

Comment:
Q: Wasn't the plan to move to a consistent `_target?: Pokemon` or if `move` etc. is required -> `_target: Pokemon | undefined`?